lib/ansible/playbook/helpers.py

```diff
@@ -207,7 +207,7 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
                     try:
                         data = loader.load_from_file(include_file)
                         if data is None:
-                            return []
+                            continue
```

Comment:
@bcoca Please can you suggest me where is the best place in the repository to put this warning about the empty file? Add it as a note in of the documentation part of [https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/utilities/logic/include.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/utilities/logic/include.py) describing that the problem affects ansible <= 2.3, is it correct?